how_to_use_env/use_variable_in_code/python.md

```diff
@@ -0,0 +1,18 @@
+Read variable TOKEN in environment:
+
+```py
+import os
+os.environ.get("TOKEN") # return the token you set earlier
+```
+
+Something like:
+
+```py
+import os
+
+TOKEN = os.environ.get("TOKEN")
```

Comment:
What’s the reason for not using `os.getenv(“TOKEN”);`?

Reply:
Cuz that's what my brain produced at 3AM